android/java/org/chromium/chrome/browser/app/helpers/ImageLoader.java

```diff
@@ -110,9 +110,8 @@ private static void downloadImage(String url, final RequestManager requestManage
                 validUrl = url;
             }
 
-            WebContentsFactory webContentsFactory = new WebContentsFactory();
             WebContents webContents =
-                    webContentsFactory.createWebContentsWithWarmRenderer(
+                    WebContentsFactory.createWebContentsWithWarmRenderer(
```

Comment:
why is this calling `createWebContentsWithWarmRenderer`?

Reply:
cc: @AlexeyBarabash

Reply:
nvm, I brought this up in the original PR